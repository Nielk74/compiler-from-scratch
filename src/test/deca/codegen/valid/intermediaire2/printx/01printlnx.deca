{
    float x = 0x1.FFFFFEp0;
    int y = 2;
    printlnx("Float : ", x);
    printlnx(0x1.0p1);
    x = 7.5;
    printlnx(x);
    printx(y);
}