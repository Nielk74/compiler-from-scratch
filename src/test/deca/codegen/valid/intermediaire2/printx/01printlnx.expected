Float : 0x1.fffffep+0
0x1p+1
0x1.ep+2
2
